system/security/shared/seclib.hpp

```diff
@@ -198,9 +228,51 @@ enum authStatus : int
+static const SecFeatureBit SUF_GetFirstName                = 0x0000000000000010;
+static const SecFeatureBit SUF_SetFirstName                = 0x0000000000000020;
+static const SecFeatureBit SUF_GetLastName                 = 0x0000000000000040;
+static const SecFeatureBit SUF_SetLastName                 = 0x0000000000000080;
+static const SecFeatureBit SUF_GetEmployeeID                 = 0x0000000000000100;
+static const SecFeatureBit SUF_SetEmployeeID                 = 0x0000000000000200;
+static const SecFeatureBit SUF_GetEmployeeNumber             = 0x0000000000000400;
+static const SecFeatureBit SUF_SetEmployeeNumber             = 0x0000000000000800;
+static const SecFeatureBit SUF_GetDistinguishedName        = 0x0000000000001000;
+static const SecFeatureBit SUF_SetDistinguishedName        = 0x0000000000002000;
+static const SecFeatureBit SUF_GetRealm                    = 0x0000000000004000;
+static const SecFeatureBit SUF_SetRealm                    = 0x0000000000008000;
+static const SecFeatureBit SUF_GetFqdn                       = 0x0000000000010000;
+static const SecFeatureBit SUF_SetFqdn                       = 0x0000000000020000;
+static const SecFeatureBit SUF_GetPeer                       = 0x0000000000040000;
+static const SecFeatureBit SUF_SetPeer                       = 0x0000000000080000;
+static const SecFeatureBit SUF_GetStatus                   = 0x0000000000100000;
+static const SecFeatureBit SUF_SetStatus                   = 0x0000000000200000;
+static const SecFeatureBit SUF_GetAuthenticateStatus       = 0x0000000000400000;
+static const SecFeatureBit SUF_SetAuthenticateStatus       = 0x0000000000800000;
+static const SecFeatureBit SUF_Credentials                   = 0x0000000001000000;
+static const SecFeatureBit SUF_GetUserID                     = 0x0000000002000000;
+static const SecFeatureBit SUF_CopyTo                        = 0x0000000004000000;
+static const SecFeatureBit SUF_GetPasswordExpiration         = 0x0000000008000000;
+static const SecFeatureBit SUF_SetPasswordExpiration       = 0x0000000010000000;
+static const SecFeatureBit SUF_GetPasswordDaysRemaining    = 0x0000000020000000;
+static const SecFeatureBit SUF_GetProperty                 = 0x0000000040000000;
+static const SecFeatureBit SUF_SetProperty                 = 0x0000000080000000;
+static const SecFeatureBit SUF_GetPropertyInt                = 0x0000000100000000;
+static const SecFeatureBit SUF_SetPropertyInt                = 0x0000000200000000;
+static const SecFeatureBit SUF_GetPropertyIterator           = 0x0000000400000000;
+static const SecFeatureBit SUF_Clone                         = 0x0000000800000000;
+static const SecFeatureBit SUF_GetDataElement              = 0x0000001000000000;
+static const SecFeatureBit SUF_GetDataElements             = 0x0000002000000000;
+static const SecFeatureSet SUF_ALL_FEATURES                = 0x0000003FFFFFFFFF; // update to include all added feature bits
```

Comment:
The alignment was done intentionally - every four rows, corresponding to every hex digit - because it was easier for me to confirm I was setting correct bits.